client/src/test/java/org/ehrbase/client/classgenerator/examples/coronaanamnesecomposition/definition/BewertungDesGesundheitsrisikosBewertungDesGesundheitsrisikosVorhandenseinDvcode_.java

```diff
@@ -6,7 +6,7 @@
 
 @Entity
 @OptionFor("DV_CODED_TEXT")
-public class BewertungDesGesundheitsrisikosVorhandenseinDvcodedtextSpezifischerRisikofaktor implements BewertungDesGesundheitsrisikosVorhandenseinChoice {
+public class BewertungDesGesundheitsrisikosBewertungDesGesundheitsrisikosVorhandenseinDvcode_ implements BewertungDesGesundheitsrisikosVorhandenseinChoice {
```

Comment:
@stefanspiska is the underscore at the end correct? seen that in other class names above.

Reply:
Yes that means that the name was shortened.